xapian-core/backends/honey/honey_compact.cc

```diff
@@ -2033,12 +2033,16 @@ HoneyDatabase::compact(Xapian::Compactor* compactor,
 	    auto db = static_cast<const GlassDatabase*>(sources[i]);
 	    auto& v_in = db->version_file;
 	    auto& v_out = version_file_out;
+	    // Glass backend doesn't track unique term bounds, hence setting
+	    // them to 0.
```

Comment:
I think this risks misleading the reader.  Someone new to this code and trying to follow it (or indeed us in several years once we've forgotten the details) would probably think this means that a converted database has zero for the bounds.

It's true we do pass zero for them here, but then we calculate them as we convert the termlist table and fill in the correct values before we write out `version_file_out`.  The part after the "but" is probably the most interesting thing to tell the reader, as it isn't obvious without careful study of the code.

Reply:
Done